lib/suggestion-list-element.js

```diff
@@ -167,24 +159,7 @@ class SuggestionListElement extends HTMLElement {
     }
 
     atom.views.updateDocument(this.renderItems.bind(this))
-    return atom.views.readDocument(this.readUIPropsFromDOM.bind(this))
-  }
-
-  addActiveClassToEditor () {
```

Comment:
I'm trying to recall why we needed to add this. Why are we able to remove it now?